haskell-org-committee.markdown

```diff
@@ -44,25 +50,47 @@ The committee consists of 7 members serving 3-year terms. The current members ar
   * Rebecca Skinner (term ends 2022)
   * Alexandre Garcia de Oliveira (term ends 2022)
```

Comment:
I skipped making updates to this list (as the dates point out an update is needed), but can apply them and that would address #62.

Reply:
Scratch that, as Tikhon notes in that ticket, the vote was delayed and an update is pending (there's no update to make right now, so ignore this thread).